.env.template

```diff
@@ -15,3 +15,5 @@ OPENAI_AZURE_DEPLOYMENT_ID=deployment-id-for-azure
 IMAGE_PROVIDER=dalle
 HUGGINGFACE_API_TOKEN=
 USE_MAC_OS_TTS=False
+RISK_THRESHOLD=0.4
+RISK_EVALUATION_MODEL=gpt-4
```

Comment:
I believe this file should hold the same default than the ones in code. Either change that one or the other to align the values.

Reply:
Thank you, that was an oversight on my part. Fixed now.